### PR TITLE
Fixes #720 - added background & padding-top to link

### LINKED ISSUE #720: Add border to "Report Bug" link/button/action
Ask users if they understand that they need to click on the report bug for opening the form.

### PR DESCRIPTION
**Pull request by zoepage**
r? @miketaylr

### CODE REVIEW

File: webcompat/static/css/development/components/hero.css [hunk 1]
```diff
@@ -82,7 +84,7 @@
         vertical-align: middle;
         margin: 0 0 0 .3em;
         background-color: #414042;
-        color: var(--Hero-colorIcon);
+        color: #ffc900;
```
Comment (miketaylr) [inline webcompat/static/css/development/components/hero.css]: nit: `var(--Hero-backgroundColor)`

File: webcompat/static/css/development/components/hero.css [hunk 2]
```diff
@@ -52,23 +52,25 @@
...
     font-size: 2em;
-    color: #414042;
     text-decoration: none;
-    transition: top 100ms ease-out 0ms;
-    top:1em;
+    top: 1em;
+    margin-top: 2.4em;
+    padding: 0.4em 0.6em;
+    border-radius: 0.3em;
+    border: 1px solid #ffc900;
```
Comment (miketaylr) [inline webcompat/static/css/development/components/hero.css]: nit: var(--Hero-backgroundColor)

File: webcompat/static/css/development/components/hero.css [hunk 3]
```diff
@@ -52,23 +52,25 @@
...
+    background: #414141;
+    color: #ffc900;
```
Comment (miketaylr) [inline webcompat/static/css/development/components/hero.css]: nit: var(--Hero-backgroundColor)

File: webcompat/static/css/development/components/hero.css [hunk 4]
```diff
@@ -52,23 +52,25 @@
...
-    color: #414042;
     text-decoration: none;
-    transition: top 100ms ease-out 0ms;
-    top:1em;
+    top: 1em;
+    margin-top: 2.4em;
+    padding: 0.4em 0.6em;
+    border-radius: 0.3em;
+    border: 1px solid #ffc900;
+    background: #414141;
```
Comment (miketaylr) [inline webcompat/static/css/development/components/hero.css]: `--base-variantColorDark` is  `#404040`, any reason to not use that?

**Comment by zoepage** [comment]
@miketaylr ?

File: webcompat/static/css/development/components/hero.css [hunk 5]
```diff
@@ -52,23 +52,25 @@
...
+    opacity: 0.95;
   }
 
       /* event */
       .wc-Hero-link:hover,
       .wc-Hero-link:focus {
-        text-decoration:none;
-        color: #fff;
-      }
+        text-decoration:none;      }
```
Comment (miketaylr) [inline webcompat/static/css/development/components/hero.css]: Can we move `}` to its own line?

**Comment by miketaylr** [comment]
The other option would be to leave the yellow bg hover change, but not remove it once the transition is done. That way there's no yellow blinking on and off.

**Comment by zoepage** [comment]
What about a yellow border around the circle? And change of background-color on hover?
Does this still feel too flashy?

**Comment by miketaylr** [comment]
> What about a yellow border around the circle? And change of background-color on hover?
> Does this still feel too flashy?

That could work! I think the main issue I have is that it toggles on when you click, then toggles off after the "open form" transition is done. It's quite literally a flash. If we can show the border (or bg) when the arrow is in the down state always, that problem goes away.

**Comment by zoepage** [comment]
@miketaylr we discussed it a bit different, but what do you think about not having a background like in the last commit? Just the border to show off the hover? Or is this still too flashy?

**Comment by miketaylr** [comment]
> Or is this still too flashy?

Would it be much work for the down arrow state to keep the circle border?

**Comment by zoepage** [comment]
Not at all. I just liked the hover state of it, but will add the border to the active state.

**Comment by zoepage** [comment]
Not sure, if you can see it, but after closing the form there is a small flash of the border. 
This happens because of a hover by accident. Pushing the state. Not sure, if fixing this would be overdoing it. 
button2

File: webcompat/static/css/development/components/hero.css [hunk 6]
```diff
@@ -52,23 +52,25 @@
     display: inline-block;
     position: relative;
     font-size: 2em;
-    color: #414042;
     text-decoration: none;
-    transition: top 100ms ease-out 0ms;
-    top:1em;
+    top: 1em;
+    margin-top: 2.4em;
+    padding: 0.4em 0.6em;
```
Comment (magsout) [inline webcompat/static/css/development/components/hero.css]: Sorry I'm not the review, but if you can remove `0`  before all `.`

`padding: .4em .6em`

**Comment by miketaylr** [comment]
Let's go ahead and land this, and see if anyone files a bug or complains. I might be over-thinking the flash stuff.

@zoepage can you make the change requested by @magsout, and then squash commits down to a single one? feel free to merge and add the "add-to-changelog" label after that!

**Comment by karlcow** [comment]
@zoepage @miketaylr `fixing` is not a working keywords. See https://help.github.com/articles/closing-issues-via-commit-messages/

**Comment by karlcow** [comment]
I modified the title of the PR

**Comment by zoepage** [comment]
@miketaylr done.

File: webcompat/static/css/development/components/hero.css (hunk 5 above)
Comment (miketaylr) [inline webcompat/static/css/development/components/hero.css]: (i took care of that in c187add54a2dc0b341af2032b2a7b1766b29c7f9)

**Comment by miketaylr** [comment]
Thanks @zoepage!

**Pull request merged**